main.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/******************* HEADER SECTION WITH NAVIGATION LIST *************************/
+.main-header {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 80px;
+  margin-right: 80px;
+  padding: 20px;
+}
+
+.main-header .Logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.bar-bar ul li a {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.bar-bar ul li a:hover {
+  font-weight: bolder;
+}
+
+.bar-bar .icon {
+  display: none;
+}
+
+/******************* MAIN SECTION AREA *************************/
+.main-section-area {
+  height: 500px;
+  background-image: url("images/Header\ bg.svg");
+  background-size: cover;
+}
+
+.main-slide {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  padding-left: 260px;
+  padding-right: 260px;
+}
+
+.slide-text {
+  display: flex;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  margin: 12px 0;
+}
+
+.connect {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media span img {
+  width: 21px;
+  height: 20px;
+  padding-top: 10px;
+  padding-right: 10px;
+}
```

Comment:
- [ ] [REQUIRED] Please, use `flexbox` to layout the social bar icons as mentioned in project requirements 🧾 .